build/prerelease.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# This will become the patch digit on the extension version. To avoid potential
+# issues, it uses a two digit year to keep the entire number well under 2^31-1.
+name: $(Date:yyMMdd)$(Rev:rrr)
+
+trigger: none
+pr: none
+schedules:
+  - cron: "0 2 * * *"
+    displayName: Daily 2 AM
+    branches:
+      include:
+        - main
+
+variables:
+  # TODO: change this to real sign after getting signing approval.
```

Comment:
I'd like to merge as-is to main (unless signing approval comes back very quickly). No publishing of test signed bits will happen because the publishing PAT isn't wired up yet, and it will give us a few days to make sure the pipeline is scheduled as expected before publishing for real.

Reply:
I think that sounds reasonable